.travis.yml

```diff
@@ -3,19 +3,16 @@ language: go
 go:
   - 1.5.4
   - 1.6.3
+  - 1.7
 
 go_import_path: google.golang.org/grpc
 
 before_install:
-  - go get golang.org/x/tools/cmd/goimports
-  - go get github.com/golang/lint/golint
-  - go get github.com/axw/gocov/gocov
-  - go get github.com/mattn/goveralls
-  - go get golang.org/x/tools/cmd/cover
+  - go get -u golang.org/x/tools/cmd/goimports github.com/golang/lint/golint github.com/axw/gocov/gocov github.com/mattn/goveralls golang.org/x/tools/cmd/cover
```

Comment:
It's only cached in Travis if you have something like this:

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/.travis.yml#L20-L23